R/legend.R

```diff
@@ -215,3 +215,24 @@ easy_add_legend_title <- function(..., teach = FALSE) {
 .all_legend_aes <- c("alpha", "cex", "col", "color", "colour",
                      "fill", "linetype", "lty", "lwd", "pch", "radius",
                      "shape", "size", "weight", "width")
+
+
+#' Easily remove legend title
+#' Remove the legend title
+#' @md
+#' @return a \code{\link[ggplot2]{theme}} object
+#' @export
+#' @examples
+#' library(ggplot2)
+#' # remove legend title from all aesthetics
+#' ggplot(mtcars, aes(wt, mpg, colour = cyl)) +
+#'   geom_point() + easy_remove_legend_title()
+
+
+easy_remove_legend_title <- function(){
```

Comment:
The `teach` argument for this one is pretty straightforward since there's no options.

```suggestion
easy_remove_legend_title <- function(teach = FALSE){
  
  if (teach) {
    message("easy_remove_legend_title call can be substituted with:")
    message(strwrap('theme(legend.title = element_blank())',
      width = 80,
      exdent = 2,
      prefix = "\n",
      initial = ""
    ))
  }
  
  ggplot2::theme(legend.title = ggplot2::element_blank())
  
}

```

Reply:
Ah sorry I wasn't aware of the teach argument.